SebastianHerrera/bootcamp-java/src/main/java/hackerrank/day2/exercise2/Operations.java

```diff
@@ -0,0 +1,11 @@
+package hackerrank.day2.exercise2;
+
+/**
+ * @author Sebastian Herrera (bermeosebas@gmail.com)
+ * @created 27/08/21
+ */
+class Operations {
```

Comment:
"1. A class named Arithmetic with a method named add that take 2 integers....."
"2. A class named Adder that inherits from....."

Operations???